examples/typescript/src/App.tsx

```diff
@@ -50,7 +50,7 @@ chatClient.connectUser({ id: userId }, userToken);
 const App = () => (
   <Chat client={chatClient}>
     <ChannelList filters={filters} sort={sort} options={options} showChannelSearch />
-    <Channel>
+    <Channel Timestamp={() => <>Lol no</>}>
```

Comment:
```suggestion
    <Channel>
```

Reply:
Sorry about that 😆 I decided to make some last minute tests, should have drafted the PR while I'm trying stuff out.